app/assets/javascripts/models/page/project_page.js.coffee

```diff
@@ -56,6 +60,16 @@ ns.ProjectPage = do (ko,
       setTimeout((=>
         @_loadFromUrl()
         $(window).on 'edsc.pagechange', @_loadFromUrl), 0)
+   
+    showType: =>
+      type = ""
```

Comment:
This is an odd method. I'd suggest setting a variable equal to `@query.serialize()` and using a standard if/else if/else statement.

Reply:
How about a compromise and I just split it up in a more eye-friendly conditional.  :)

The previous code was due to a mild aversion to having multiple returns in a function - however, I agree that it 'looked funny'.  I did keep @query.serialize (seems unnecessary to tuck into into a variable when it's only used a few times), however, the conditional structure should be easier on the eyes.